python_modules/dagster/dagster/_core/definitions/time_window_partition_mapping.py

```diff
@@ -16,6 +16,25 @@
 from dagster._serdes import whitelist_for_serdes
 
 
+class MappedTimeWindowPartitionsResult(
```

Comment:
A more compact notation here is:

```
class MappedTimeWindowPartitionsResult(NamedTuple):
    """Docstring"""
    partitions_subset: TimeWindowPartitionsSubset
    mapped_to_invalid_time_windows: bool
```

I think it also would be reasonable to keep the returned tuple "unnamed", i.e. _map_partitions returns a regular tuple (subset, mapped_to_invalid), and the callsites of _map_partitions would just have:

partitions_subset, mapped_to_invalid_time_windows = _map_partitions(...)

I'll leave that bit up to you though

Reply:
good call, I've updated to the suggested notation. I prefer to still use the named tuple to keep the return objects unambiguous